File: cash.py
#import get_float
from cs50 import get_float
# money that we owed, cent(value of coins), count(number of coins)
change = 0.00
# a loop for get the change from someone
while change == 0:
    change = get_float("owed change: ")
    count = 0
    # cent is an int value so we should round the change
    cent = round(change * 100)
# a loop for he 25, 10, 5, 1 cent
while count == 0:
    while cent >= 25:
        cent -= 25
        count += 1

    while cent >= 10:
        cent -= 10
        count += 1

    while cent >= 5:
        cent -= 5
        count += 1

    while cent >= 1:
        cent -= 1
        count += 1
# print number of coins that we gave
if cent == 0:
    print("counted coins: ", count)

File: cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    //money that we owed
    float dollar = 0;

    //get the value of owed money from user
    do
    {
        dollar = get_float("owed money: ");
    }
    //check if owed money is true
    while (dollar < 0);

    //count number of coin and cent of money that we owed
    int count = 0;
    int cent = round(dollar * 100);

    //make 25, 10, 5, 1 cent and count them
    while (count == 0)
    {
        //25 cent coin
        while (cent >= 25)
        {
            cent -= 25;
            count++;
        }

        //10 cent coin
        while (cent >= 10)
        {
            cent -= 10;
            count++;
        }

        //5 cent coin
        while (cent >= 5)
        {
            cent -= 5;
            count++;
        }

        //1 cent coins
        while (cent >= 1)
        {
            cent -= 1;
            count++;
        }

        if (cent == 0)
        {
            //print number of coins (count)
            printf("number of rounded coin that i gave you: %i\n", count);
        }
    }
}
